tests/main.py

```diff
@@ -185,12 +190,16 @@ def test_feed_by_new():
     feed = etree.fromstring(response.text.encode("utf-8"))
     expected = _get_expected_xml_result("test_feed_by_new")
 
-    # Remove runtime-generated dates
+    # Remove runtime-generated date and title because encoding is hard
```

Comment:
I don't think this will be understandable/useful. What's actually happening here/the problem?

Reply:
Every year or so something within pytest/feedgen/whatever combo changes and either escapes the long dash `–` or not. I don't wish to spend time on that as it's not the point of that test anyway.

Reply:
Well, that's probably enough documentation, for anybody that actually want's to find it. ;)